diff --git a/vendor/wikimedia/parsoid/src/Wt2Html/PP/Processors/AddMediaInfo.php b/vendor/wikimedia/parsoid/src/Wt2Html/PP/Processors/AddMediaInfo.php
index 507ae1d..77c28fd 100644
--- a/vendor/wikimedia/parsoid/src/Wt2Html/PP/Processors/AddMediaInfo.php
+++ b/vendor/wikimedia/parsoid/src/Wt2Html/PP/Processors/AddMediaInfo.php
@@ -189,7 +189,7 @@ class AddMediaInfo implements Wt2HtmlDOMProcessor {
 				$source->setAttribute( 'data' . $fromFile . '-width', (string)$o['width'] );
 				$source->setAttribute( 'data' . $fromFile . '-height', (string)$o['height'] );
 			}
-			if ( $dataFromTMH ) {
+			if ( $dataFromTMH && isset( $o['title'] ) && isset( $o['shorttitle'] ) ) {
 				$source->setAttribute( 'data-title', $o['title'] );
 				$source->setAttribute( 'data-shorttitle', $o['shorttitle'] );
 			}
